src/actions/users.js

```diff
@@ -610,6 +610,37 @@ export function getMe() {
     };
 }
 
+export function updateServiceTermsStatus(serviceTermsId, accepted) {
+    return async (dispatch, getState) => {
+        const {data, error} = await dispatch(bindClientFunc(
+            Client4.updateServiceTermsStatus,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_REQUEST,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_SUCCESS,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_FAILURE,
+            serviceTermsId,
+            accepted
+        ));
+        if (data && data.status === 'OK') {
```

Comment:
can we make 'OK' a constant? is there any other way to check for success? like a `status_code`?